samplers/samplers.go

```diff
@@ -144,26 +178,26 @@ func (c *Counter) Sample(sample float64, sampleRate float32) {
 
 // Flush generates a DDMetric from the current state of this Counter.
 func (c *Counter) Flush(interval time.Duration) []DDMetric {
-	tags := make([]string, len(c.tags))
-	copy(tags, c.tags)
+	Tags := make([]string, len(c.Tags))
```

Comment:
looks like a bunch of these got capitalized by accident when we were cleaning up the refactor... not really important to fix but does look odd, since we decapitalized some of them but not all